src/apps/lwaftr/address_map.lua

```diff
@@ -272,6 +285,77 @@ function compile(file)
    return attach_lookup_helper(builder:build())
 end
 
+local verbose = os.getenv('SNABB_LWAFTR_VERBOSE') or true
```

Comment:
this "or true" is just a temporary thing and probably shouldn't be there.  probably we should put a log function in the utils, unless snabb gives us something nice already